source/tyr/tests/integration/datapost_test.py

```diff
@@ -32,12 +41,56 @@ def test_post_pbf(create_instance_fr):
     os.remove('/tmp/empty_pbf.osm.pbf')
     assert (not os.path.isfile('/tmp/empty_pbf.osm.pbf'))
 
+
 def test_post_no_file(create_instance_fr):
     tester = app.test_client()
     resp = tester.post('/v0/jobs/fr')
     assert resp.status_code == 404
 
+
 def test_post_bad_instance(create_instance_fr):
     tester = app.test_client()
     resp = tester.post('/v0/jobs/fr_ko')
     assert resp.status_code == 404
+
+
+def test_post_pbf_autocomplete(create_instance_fr):
+    with app.app_context():
+        ac_path = current_app.config['TYR_AUTOCOMPLETE_DIR']
+        assert not os.path.exists(ac_path)
+        filename = 'empty_pbf.osm.pbf'
+        path = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))), 'tests/fixtures/', filename)
+        with open(path, 'rb') as f:
+            files = {'file': (f, filename)}
+            json_response = api_post('/v0/autocomplete_parameters/fr/update_data', data=files)
+            assert 'job' in json_response
+            job = json_response['job']
+            assert 'id' in job
+            assert type(job['id']) == int
+
+            jobs = get_jobs_from_db()
+            assert len(jobs) == 1
+            assert jobs[0].id == job['id']
+            shutil.rmtree(ac_path)#Remove autocomplete directory
+
+
+def test_post_pbf_autocomplete_without_files(create_instance_fr):
+    with app.app_context():
+        ac_path = current_app.config['TYR_AUTOCOMPLETE_DIR']
+        assert not os.path.exists(ac_path)
+        filename = 'empty_pbf.osm.pbf'
+        path = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))), 'tests/fixtures/', filename)
+        with open(path, 'rb') as f:
+            response = api_post('/v0/autocomplete_parameters/fr/update_data', check=False)
+            assert response[1] == 404
```

Comment:
I think it's clearer to deconstruct the tuple
`response, status = api_post('/v0/autocomplete_parameters/fr/update_data', check=False)`